src/main/java/com/amazon/ion/impl/IonWriter_1_1.kt

```diff
@@ -149,16 +148,13 @@ interface IonWriter_1_1 {
 
     fun writeBool(value: Boolean)
 
-    fun writeInt(value: Byte)
-    fun writeInt(value: Int)
```

Comment:
🗺️  The implementations of these ended up just casting `value` to a `Long` and then calling the existing implementation for writing a `Long`. If we want to have optimized implementations for smaller integer sizes, we can add that later.